Utils/XDrive.hpp

```diff
@@ -31,22 +31,22 @@ class XDriveRobot {
     vex::inertial& inertialSensor;
 
     vex::motor& flywheel;
-    vex::motor& indexer;
+    vex::motor& expander;
     vex::motor& intake;
+    vex::motor& roller;
 
     void calibrate () {
         inertialSensor.calibrate();
         vex::task::sleep(1500);
     }
 
-    void move(double blspeed, double brspeed, double flspeed, double frspeed, RollingScreen& rs) {
+    void move(double blspeed, double brspeed, double flspeed, double frspeed) {
```

Comment:
please rename this method -- move() is a very powerful C++ function which we don't intend to collide with its name (even though you can as our move() is within XDrive class).
More important reason: the function does not move your robot, it only set motor speed (as a convenience function). So better call it setSpeed() or sth similar.